README.md

```diff
@@ -199,9 +199,11 @@ class User < ActiveRecord::Base
 
   def self.scim_queryable_attributes
     return {
-      givenName:  :first_name,
-      familyName: :last_name,
-      emails:     :work_email_address,
+      givenName:        { column: :first_name },
+      familyName:       { column: :last_name },
+      emails:           { column: :work_email_address },
+      "groups":         { column: MockGroup.arel_table[:id] },
+      "groups.value":   { column: MockGroup.arel_table[:id] },
```

Comment:
It's good to include something in an example but in this case it's quite an advanced jump and has no explanation of why it's there or what it means, especially without a clear idea of MockGroup. Note the preamble at line 115, which points to the RDoc format (note RDoc, not Markdown) comments in the source code at

https://github.com/RIPAGlobal/scimitar/blob/main/app/models/scimitar/resources/mixin.rb

...so perhaps that's the place to explain why and how AREL table references can be used?